src/has.ts

```diff
@@ -64,6 +64,7 @@ add('dom-mutationobserver', function(): boolean {
 add('microtasks', function () {
 	return has('promise') || has('host-node') || has('dom-mutationobserver');
 });
+add('object-assign', typeof (<any> Object).assign === 'function');
```

Comment:
We should solidify the `has` naming...  I know somewhere else we discussed that we would prefer to use `es6-object-assign` for something like this (and `object-observe` should be [in theory] `es7-object-observe` and `promise` should be `es6-promise`).